README.md

```diff
@@ -265,6 +265,7 @@ See the [API documentation](./docs/api.md) for all endpoints.
 
 ### Web & Desktop
 
+- [IntelliBar](https://intellibar.app/)
```

Comment:
Possible to add this further down in the list?

Reply:
Sure, done.